lib/backend/repositories/user_repo.dart

```diff
@@ -229,6 +230,17 @@ class UserRepo {
     return false;
   }
 
+  Future<void> deleteUserAccount(String userAddress, String password) {
+    if (userAddress.isNotEmpty &&
+        userAddress != null &&
```

Comment:
if it's not empty then it's not null (the order of comparison matters)

Reply:
Right, good catch.